src/ssl.c

```diff
@@ -31185,6 +31421,34 @@ int wolfSSL_EC_GROUP_get_order(const WOLFSSL_EC_GROUP *group,
 
     return WOLFSSL_SUCCESS;
 }
+
+int wolfSSL_EC_GROUP_order_bits(const WOLFSSL_EC_GROUP *group)
+{
+    int ret;
+    int order_bits = 0;
+    mp_int order;
+
+    if (group == NULL || group->curve_idx < 0) {
+        WOLFSSL_MSG("wolfSSL_EC_GROUP_order_bits NULL error");
+        return 0;
+    }
+
+    ret = mp_init(&order);
+    if (ret == 0) {
+        ret = mp_read_radix(&order, ecc_sets[group->curve_idx].order,
+            MP_RADIX_HEX);
+        if (ret == 0)
+            ret = mp_unsigned_bin_size(&order);
+        if (ret >= 0)
+            ret = mp_to_unsigned_bin(&order, (byte*)&order_bits);
```

Comment:
This is big endian unsigned format right?  Is that what the caller is expecting?  Did you look at mp_count_bits()?